frontend/public/components/machine.tsx

```diff
@@ -162,13 +170,24 @@ const MachineDetails: React.SFC<MachineDetailsProps> = ({ obj }: { obj: MachineK
   );
 };
 
-export const MachineList: React.SFC = (props) => {
+type MachineListProps = {
+  data: MachineKind[];
+  loaded: boolean;
+  loadError: any;
+};
+
+export const MachineList: React.FC<MachineListProps> = (props) => {
   const { t } = useTranslation();
-  const MachineTableHeader = () => {
-    return [
+
+  const machineTableColumn = React.useMemo<TableColumn<MachineKind>[]>(
+    () => [
       {
         title: t('public~Name'),
-        sortField: 'metadata.name',
+        sort: (data, direction) => {
```

Comment:
I'm unclear on why this was changed to `sort` whereas the other columns still use `sortField`

Reply:
I forgot to do that. All other columns are changed now as well